packages/legend-application-repl/src/stores/dataCube/core/DataCubeQuerySnapshot.ts

```diff
@@ -180,12 +189,14 @@ export function createSnapshot(
     filter: undefined,
     renamedColumns: [],
     groupByColumns: [],
+    groupByExpandedKeys: [],
```

Comment:
I don't agree with this change, to me, the snapshot is 1-1 to the actual query, here you're hacking it as the vessel to build the intermediate query for drilldown data